models/dojo.py

```diff
@@ -65,7 +80,7 @@ def add_person(self,arg):
             if self.full_name not in Dojo.staff:
                 new_staff = Staff(self.fname,self.lname)
                 Dojo.staff[self.full_name] = {}
```

Comment:
A better way to do this is to save the staff or fellow details in form of an object and not a dictionary. e.g `Dojo.staff[new_staff.full_name] = new_staff` or `Dojo.staff[new_staff.id] = new_staff`. With this, you wouldn't need lines 83-86 and line 101-104.